BeeJet.Bot/BeeJet.Bot.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
Kunnen we al wel naar .net 7.0? Ik dacht dat IGDB daar een probleem mee had

Reply:
Yes, heb het getest en opeens werkt het wel